src/components/pages/ProductionQuota.vue

```diff
@@ -233,7 +311,7 @@ export default {
     getCurrentPerson() {
       const personId = this.$route.params.person_id
       if (personId && this.personMap) {
-        return this.personMap.get(personId)
+        return personMap.get(personId)
```

Comment:
Inconsistent test (`this.personMap`) vs usage (`personMap`)

Reply:
I don't understand your comment.